File: hw/suarez/U3/HW17Singleton/src/ec/edu/espe/view/Calculator.java
package ec.edu.espe.view;
import ec.edu.espe.model.USTax;

/**
 *
 * @author Bernardo
 */

public class Calculator { //add "public" and capitalize class name
    public static void main(String[] args) {
        USTax tax = USTax.getInstance();

        float basePrice = 100.00f;
        float total = tax.salesTotal(basePrice);

        System.out.println("Base price --> $" + basePrice);
        System.out.println("Tax Rate --> " + tax.getTaxRate() * 100 + "%");
        System.out.println("Total price --> $" + total);
    }
}

File: hw/fuentes/U3/HW18SIGNLE/HW18SINGLEPY/ec/edu/espe/ustax/calculator.py
from model.ustax import USTax

def main():
    tax = USTax()
    price = float(input("Enter the product price: "))
    total_price = tax.CalculatesalesTotal(price)
    print(f"The total price with tax is: {total_price}")

if __name__ == "__main__":
    main()